lib/audiofile.py

```diff
@@ -49,3 +49,26 @@ def read(filePath, makeMono=False):
 def openWithFfmepg(filePath):
     # TODO finish this shit
     cmd = f'ffmpeg -i {filePath}'
+
+
+def convertMonoDatatoBytes(x, rate, filename):
+    """ Convert synthetic signal x to bytes object readable by readframes function of wave module
+    Args:
+        [list]:     x,          signal data
```

Comment:
I prefer explicit variable name (and if possible more than 1 letter), except if for acoustic engineer it's a standard